src/styles/base/index.scss

```diff
@@ -1,5 +1,24 @@
+* {
```

Comment:
this will clean up all margins for all elements, you don't need it, you have to clean up margins only for body

Reply:
done